docs/user/icl/ssh-add-key-to-artemis.rst

```diff
@@ -0,0 +1,66 @@
+.. _use ssh key:
+
+Using SSH with Artemis
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Content of this document
+    :local:
+    :depth: 1
+
+You can use SSH keys to establish a secure connection between your computer and Artemis when you are performing Git operations (pull, clone, push) from your local machine.
+Personal keys are attached to your Artemis account – they are bound by that account's permissions and use the account's identity for any operations.
+To use your generated SSH keys with Artemis, you need to add it in the account Settings.
+
+
+Add an SSH key to your Artemis account
+""""""""""""""""""""""""""""""""""""""
+
+**1. Copy your public key**
+
+On Windows in your command prompt, change directory to your .ssh directory, and copy the public key file to your clipboard by running:
+
+.. code-block:: bash
+
+    cd %userprofile%/.ssh
+    clip < id_ed25519.pub
+
+On macOS or Linux simply run the following in a terminal:
+
+.. code-block:: bash
+
+    pbcopy < ~/.ssh/id_ed25519.pub
+
+If pbcopy isn't working, locate the hidden .ssh folder, open the file in a text editor, and copy it to your clipboard.
+Note that on Linux, you may need to download and install xclip, then use that, as shown in this code snippet:
```

Comment:
```suggestion
Note that on Linux, you may need to download and install `xclip`, then use that, as shown in this code snippet:
```
Maybe we can also link to `xclip`? https://wiki.ubuntuusers.de/xclip/ or https://opensource.com/article/19/7/xclip

Reply:
good idea